sites/public/src/components/shared/FormSummaryDetails.tsx

```diff
@@ -470,7 +474,12 @@ const FormSummaryDetails = ({
               label={t("t.income")}
               className={"pb-4"}
             >
-              ${application.income} {t(`t.${application.incomePeriod}`)}
+              $
```

Comment:
This is an unrelated bug fix - we weren't showing commas or decimals correctly when displaying the income value